test/i-frame.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { browser } from 'protractor';
+import { IFramePage } from '../src/page';
+
+describe('Given a SDET learning protractor', () => {
+  const iFrame: IFramePage = new IFramePage();
+  describe('and then open Iframe page', () => {
```

Comment:
```suggestion
  describe('When open Iframe page', () => {
```
Remember Gerkhin syntax is Given => When => Then